### PR TITLE
Updated log handler to watcher

### PR DESCRIPTION
**Pull request by SinghDivneet**
ADO-3132

### CODE REVIEW

File: chatbot_ner/config.py [hunk 1]
```diff
@@ -32,7 +32,8 @@
 ner_logger = logging.getLogger('NERLogger')
 ner_logger.setLevel(LOG_LEVEL)
 # Add the log message handler to the logger
-handler = logging.handlers.RotatingFileHandler(NER_LOG_FILENAME, maxBytes=10 * 1024 * 1024, backupCount=5)
+handler = logging.handlers.WatchedFileHandler
+#handler = logging.handlers.RotatingFileHandler(NER_LOG_FILENAME, maxBytes=10 * 1024 * 1024, backupCount=5)
```
Comment (srajdev) [inline chatbot_ner/config.py]: E265 block comment should start with '# '

File: chatbot_ner/config.py [hunk 2]
```diff
@@ -43,7 +44,8 @@
 nlp_logger = logging.getLogger('NLPLibLogger')
 nlp_logger.setLevel(LOG_LEVEL)
 # Add the log message handler to the logger
-handler = logging.handlers.RotatingFileHandler(NLP_LIB_LOG_FILENAME, maxBytes=10 * 1024 * 1024, backupCount=5)
+handler = logging.handlers.WatchedFileHandler
+#handler = logging.handlers.RotatingFileHandler(NLP_LIB_LOG_FILENAME, maxBytes=10 * 1024 * 1024, backupCount=5)
```
Comment (srajdev) [inline chatbot_ner/config.py]: E265 block comment should start with '# '

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Pull request merged**